Changelog.md

```diff
@@ -2,7 +2,13 @@
 
 ### Gazebo Physics 6.x.x (202X-XX-XX)
 
-### Gazebo Physics 6.3.0 (2023-01-20)
+### Gazebo Physics 6.3.0 (2023-02-02)
+
+1. Port: 5 to 6
+    * [Pull request #477](https://github.com/gazebosim/gz-physics/pull/477)
```

Comment:
I don't usually include the merge-forward pull requests in the changelog; I prefer to mention the pull requests that were merged forward. I think the material changes ported forward are from https://github.com/gazebosim/gz-physics/pull/400 and https://github.com/gazebosim/gz-physics/pull/473